recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge/label/numpy_rc,conda-forge
```

Comment:
It looks like these wound up on `main`

<img width="1840" alt="Screenshot 2024-05-01 at 9 13 38 PM" src="https://github.com/conda-forge/numpy-feedstock/assets/3019665/c6ba63b2-3bcf-4f27-ab54-3eb5c217d487">

Think we wanted

```suggestion
channel_targets:
- conda-forge numpy_rc
```

Reply:
Have manually moved them over to the `numpy_rc` label. Though please recheck

<img width="1840" alt="Screenshot 2024-05-01 at 9 16 31 PM" src="https://github.com/conda-forge/numpy-feedstock/assets/3019665/24fab0e6-21da-4835-935a-43e5870b1f07">

Reply:
Sorry I misunderstood. Have reverted this as explained in comment: https://github.com/conda-forge/numpy-feedstock/pull/314/files#r1587043959

Reply:
Indeed, this was fully intentional (as well as documented in various places and referenced in the OP). Please give some time for others to react before you take unilateral action.

I did double check, and all the new numpy builds are on `main`, as they should be.